src/module.ts

```diff
@@ -386,38 +387,38 @@ export enum BinaryOp {
   MaxI16x8 = 143 /* _BinaryenMaxSVecI16x8 */,
   MaxU16x8 = 144 /* _BinaryenMaxUVecI16x8 */,
   AvgrU16x8 = 145 /* _BinaryenAvgrUVecI16x8 */,
-  AddI32x4 = 146 /* _BinaryenAddVecI32x4 */,
-  SubI32x4 = 147 /* _BinaryenSubVecI32x4 */,
-  MulI32x4 = 148 /* _BinaryenMulVecI32x4 */,
-  MinI32x4 = 149 /* _BinaryenMinSVecI32x4 */,
-  MinU32x4 = 150 /* _BinaryenMinUVecI32x4 */,
-  MaxI32x4 = 151 /* _BinaryenMaxSVecI32x4 */,
-  MaxU32x4 = 152 /* _BinaryenMaxUVecI32x4 */,
-  DotI16x8 = 153 /* _BinaryenDotSVecI16x8ToVecI32x4 */,
-  AddI64x2 = 154 /* _BinaryenAddVecI64x2 */,
-  SubI64x2 = 155 /* _BinaryenSubVecI64x2 */,
-
-  AddF32x4 = 157 /* _BinaryenAddVecF32x4 */,
-  SubF32x4 = 158 /* _BinaryenSubVecF32x4 */,
-  MulF32x4 = 159 /* _BinaryenMulVecF32x4 */,
-  DivF32x4 = 160 /* _BinaryenDivVecF32x4 */,
-  MinF32x4 = 161 /* _BinaryenMinVecF32x4 */,
-  MaxF32x4 = 162 /* _BinaryenMaxVecF32x4 */,
-  PminF32x4 = 163 /* _BinaryenPMinVecF32x4 */,
-  PmaxF32x4 = 164 /* _BinaryenPMaxVecF32x4 */,
-  AddF64x2 = 165 /* _BinaryenAddVecF64x2 */,
-  SubF64x2 = 166 /* _BinaryenSubVecF64x2 */,
-  MulF64x2 = 167 /* _BinaryenMulVecF64x2 */,
-  DivF64x2 = 168 /* _BinaryenDivVecF64x2 */,
-  MinF64x2 = 169 /* _BinaryenMinVecF64x2 */,
-  MaxF64x2 = 170 /* _BinaryenMaxVecF64x2 */,
-  PminF64x2 = 171 /* _BinaryenPMinVecF64x2 */,
-  PmaxF64x2 = 172 /* _BinaryenPMaxVecF64x2 */,
-  NarrowI16x8ToI8x16 = 173 /* _BinaryenNarrowSVecI16x8ToVecI8x16 */,
-  NarrowU16x8ToU8x16 = 174 /* _BinaryenNarrowUVecI16x8ToVecI8x16 */,
-  NarrowI32x4ToI16x8 = 175 /* _BinaryenNarrowSVecI32x4ToVecI16x8 */,
-  NarrowU32x4ToU16x8 = 176 /* _BinaryenNarrowUVecI32x4ToVecI16x8 */,
-  SwizzleV8x16 = 177 /* _BinaryenSwizzleVec8x16 */
+  AddI32x4 = 151 /* _BinaryenAddVecI32x4 */,
```

Comment:
Lot's of...

Reply:
These seem to be: https://github.com/WebAssembly/binaryen/blob/68294338a1cc7337e808671e75933b1134d18a90/src/wasm.h#L396-L400

```
  Q15MulrSatSVecI16x8,
  ExtMulLowSVecI16x8,
  ExtMulHighSVecI16x8,
  ExtMulLowUVecI16x8,
  ExtMulHighUVecI16x8,
```

Reply:
Done